build.sbt

```diff
@@ -1,9 +1,10 @@
+import Scalaz._
 
 lazy val root = Project(
   id = "root",
   base = file(".")
 ).settings(
-  scalaVersion := "2.12.2"
+  scalaVersion := "2.12.3"
```

Comment:
should we maybe create a `project/ProjectPlugin.scala` to allow use of `projectSettings`?